ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/dao/PostgresLedgerDao.scala

```diff
@@ -1073,15 +1074,15 @@ private class PostgresLedgerDao(
                     "package" -> p._1.toByteArray
                 )
               )
-            executeBatchSql(
-              SQL_INSERT_PACKAGE,
-              namedPackageParams
-            )
-            PersistenceResponse.Ok
-          }.recover {
-            case NonFatal(e) if e.getMessage.contains("duplicate key") =>
```

Comment:
I really didn't like this, the whole logic relies on the error message as a _stable API_. Good riddance!